sdks/java/io/cassandra/src/main/java/org/apache/beam/sdk/io/cassandra/CassandraService.java

```diff
@@ -45,4 +45,22 @@
   List<BoundedSource<T>> split(CassandraIO.Read<T> spec,
                                           long desiredBundleSizeBytes);
 
+  /**
+   * Create a {@link Writer} that writes entities into the Cassandra instance.
+   */
+  Writer createWriter(CassandraIO.Write<T> spec) throws Exception;
+
+  /**
+   * Writer for an entity.
+   */
+  interface Writer<T> {
+
+    void start();
```

Comment:
Why do you need start() rather than having createWriter() return an already-started writer?